### ISSUE TITLE
Hammering Sound when open Nei using with Gregtech 

### ISSUE THREAD

**Issue by Dream-Master** [rwtema/extrautilities_old_issues#687]
I using Gregtech 5.08.25 and Extra utilities together and get some wired hammer sound(from Gregtech) everytime when i open and close nei. Did you change any thing between version 1.2.7 and 1.2.8. From Time to time i get this Crash.

```
---- Minecraft Crash Report ----
// I just don't know what went wrong :(

Time: 7/19/15 4:18 PM
Description: Unexpected error

java.util.ConcurrentModificationException
    at com.google.common.collect.HashBiMap$Itr.checkForConcurrentModification(HashBiMap.java:339)
    at com.google.common.collect.HashBiMap$Itr.hasNext(HashBiMap.java:345)
    at net.minecraft.client.audio.SoundManager.func_148605_d(SoundManager.java:212)
    at net.minecraft.client.audio.SoundHandler.func_73660_a(SourceFile:179)
    at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:2013)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

```

**Comment by SvEgiiVEteR** [comment]
exactly the same problem ((

**Comment by Dream-Master** [comment]
With 1.27 i have the hammering sounds too.

**Comment by Dream-Master** [comment]
https://github.com/rwtema/extrautilities/issues/671

**Comment by Kehaan** [comment]
Same issue here

**Comment by Dream-Master** [comment]
I think the Hammering sound is the vanilla Anvil Sound not the Gregtech Sound many People think.

**Comment by Dream-Master** [comment]
@rwtema

The hammering sound not disappear with Version 1.2.9.

Please can you look inside this and if it is not on your end maybe you can say wich mod is cause. With Version 1.2.6 everything is fine.

**Comment by Nullga** [comment]
My case is mixed sound of random.break, random.anvil_use, random.anvil_break (all vanilla) and tools.drill.DrillSoft (IC2).
Confirmed by making a resource pack to disable these sounds for now however it still crashes sometimes.

**Comment by Dream-Master** [comment]
@Nullga 
Can you share this resourcepack please.

**Comment by Nullga** [comment]
I assume posting a link directly to my file isn't allowed here so try to make one yourself by following the description below. It shouldn't be tough to do. I hope the issue gets fixed sooner.

http://pastebin.com/GtmtRKgS

**Comment by Dream-Master** [comment]
Thanks i will try.

Btw where are the minecraft soundfile when i play with Techic launcher ot AT Launcher ?

**Comment by Dream-Master** [comment]
Not fixed in Version 1.2.10.

@rwtema can you please fix. Thanks

**Comment by Dream-Master** [comment]
Not fixed in Version 1.2.11.

@rwtema can you please fix. Thanks

**Comment by plaguewolf** [comment]
@Nullga 
your paste is taken down, any chance you could share the method you made those files?

this bug has just about given me a permanet eye twitch and homicidal tendencies.

**Comment by Dream-Master** [comment]
@plaguewolf 

```
+[directry]
 |  +[assets]
 |   |  +[ic2]
 |   |   |+[sounds]
 |   |   |  +[ms]
 |   |   |   |(ms.ogg)
 |   |   |(sounds.json)
 |   |  +[minecraft]
 |   |   |+[sounds]
 |   |   |  +[ms]
 |   |   |   |(ms.ogg)
 |   |   |-sounds.json
 |   |(pack.mcmeta)
 |   |(pack.png)


[square brackets] mean directlies. (round brackets) is actual files. Edit each file by copying and pasting following format. No need of equals "===" to be included.
(pack.png) shouldn't be necessary. (ms.ogg) is any kinds of mute sound file.
However, there should be a way to lower the volume instead of replacing them with a mute sound though I'm not doing it this time. Good luck!



1. (pack.mcmeta)
=========================================================
{
  "pack": {
    "pack_format": 1,
    "description": "\u00A72MUTE SOUNDS"
  }
}
=========================================================


2. (sounds.json) in [ic2] folder
=========================================================
{
  "tools.drill.DrillHard":{
    "replace":true,
    "category":"master",
    "sounds":[
      {
        "name":"ms/ms",
        "stream":false

      }]

  },

    "tools.drill.DrillSoft":{
    "replace":true,
    "category":"master",
    "sounds":[
      {
        "name":"ms/ms",
        "stream":false

      }]

  }


}
=========================================================


3. (sounds.json) in [minecraft] folder
=========================================================
{
  "random.break": {
    "replace":true,
    "category":"player",
    "sounds":[
      {
        "name":"ms/ms",
        "stream":false
      }]

  },
    "random.anvil_use": {
    "replace":true,
    "category":"block",
    "sounds":[
      {
        "name":"ms/ms",
        "stream":false
      }]

  },
      "random.anvil_break": {
    "replace":true,
    "category":"block",
    "sounds":[
      {
        "name":"ms/ms",
        "stream":false
      }]

  }  

}
=========================================================
```

**Comment by Dream-Master** [comment]
I not can fixed it with this because i not finding the minecraft audio files when using Technic Launcher or ATLauncher.

**Comment by Dream-Master** [comment]
Have to pack this in a resource pack and load it in the main menu.
It is not a fix, it is a duct tape hacky solution.

**Comment by plaguewolf** [comment]
indeed. has rwtema abandoned exU? or maybe joined the "upgrading to 1.8, so i dont care about / cant be bothered, 1.7.10 bandwagon?

**Comment by Dream-Master** [comment]
Fixed in Extra Utilities 1.2.12 and newer GT Version.
